### ISSUE TITLE
Unable to find Screen Capture option

### ISSUE THREAD

**Issue by Quixote1** [stringandstickytape/MaxLights#5]
Hi,
I was happy to find this app after trying MaxLifx years ago and not being able to successfully find my lights on my network. This new and improved version does find lights (though not all of them ever at the same time for some reason -- but I have a feeling it's the bulbs that are at fault here).

One thing that is confusing me is that I can't seem to find any screen capture option in the context menu, or anywhere else for that matter. Am I missing something or is this a bug?
Thanks!

**Comment by kcinickgx** [comment]
![2024-06-05_152513](https://github.com/stringandstickytape/MaxLights/assets/35707223/08ee3d3b-8f7a-41a6-a054-2a0725a3473a)

**Comment by Quixote1** [comment]
Thank you for taking the time to point that out to me. I can see now that I have some sort of windows/browser scaling issues (probably because I use a 4k television as my main display, I'm guessing). My sub context menu that you referenced is cut off at "Number to List", and I have no way of displaying the full menu on my screen.

**Comment by kcinickgx** [comment]
yes, it's a long list, try clicking on the very top of the page.

**Comment by Quixote1** [comment]
No such luck!
![Screencap](https://github.com/stringandstickytape/MaxLights/assets/25068715/be92b2b4-9b08-456c-b7a4-8e08d6e451bf)

**Comment by kcinickgx** [comment]
there's a graph.html with the main executable, try this, edit the file with notepad or any other text editor, search this section:
`.context-menu > .item, .context-menu > .item > .subitems > .item, .context-menu > .search, .context-menu > .search > input {
            background-color: white !important;
            color: black !important;
            font-family: sans-serif !important;
            font-size: 10px !important;
            border: 1px solid black !important;
            border-radius: 0px !important;
            font-weight:normal !important;
        }
`

replace the 10px on font-size to like 8px, then reload the page, if 8px still too big, try 6px

reload the page with F5 to the changes get in effect

**Comment by Quixote1** [comment]
Super!  I had to drop all the way down to 5px, but I now have all the list items and they are still legible!  Thanks!
Last thing, if I may -- I have another scaling issue with the list of lights in the main executable that cuts off half of the text and buttons. Do you know where I might find that font size so that I can drop that one as well?
![Screencap](https://github.com/stringandstickytape/MaxLights/assets/25068715/9e716f3d-2d90-4f63-a705-56b818476b67)

**Comment by kcinickgx** [comment]
that i don't know, it's the program itself, i think recompiling the sources, but it's a hole new level for me, sorry. maybe changing the dpi on the whole system will fix it, but everything else it's gonna be too small, i think for your screenshot you have a 150% dpi or more, try lowering to 125% 
![image](https://github.com/stringandstickytape/MaxLights/assets/35707223/3e804510-f7f6-4017-92ed-cbb6c47f28b1)

**Comment by Quixote1** [comment]
Thanks for the pointer. I wasn't keen to change the system wide settings, but thanks to your advice I played around with the properties of the executable and changed the settings in the "Change high DPI settings" on the Compatibility tab, which remedied the problem.
Thanks for all of your help!

**Comment by kcinickgx** [comment]
no problem, cheers!
